### PR TITLE
179 describe build and run in readme

### PR DESCRIPTION
**Pull request by utgarda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,49 @@
 ETHOnline 2022 entry by ChainHackers
 https://gamejutsu.app/
 
+# Run locally
+```shell
+yarn install
+```
+add the file ```.env.local``` to the root of the project and in it declare the variable ```NEXT_PUBLIC_ALCHEMY_API_KEY with the key from ALCHEMY_API```
```
Comment (utgarda) [inline README.md:9]: This needs a more explicit example - i.e. "put a line like the following in your `.env.local`
```properties
NEXT_PUBLIC_ALCHEMY_API_KEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Most often keys are same-length strings, Alchemy keys are no different. This way it's more obvious.

For single-line formatting, single backticks suffice.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,49 @@
...
+
+```shell
+yarn generate
+yarn dev
+```
+
+### Development
+```shell
+yarn install
```
Comment (utgarda) [inline README.md:18]: Is this section different from "Run locally"?

File: README.md [hunk 3]
```diff
@@ -2,6 +2,49 @@
...
+git tag v1.1.20
+```
+
+Push origin
+```shell
+git push origin v1.1.20
+```
+
+###(up to u) Docker
+???
```
Comment (utgarda) [inline README.md:46]: Please don't leave the ??? if you don't complete this one, just drop it :)

**Review by utgarda — COMMENTED** [review]
Some more cleanup please.

File: README.md (hunk 1 above)
Comment (habdevs) [reply README.md:9]: corrected

File: README.md (hunk 2 above)
Comment (habdevs) [reply README.md:18]: I don't think so. deleted

File: README.md (hunk 3 above)
Comment (habdevs) [reply README.md:46]: well deleted

**Review by habdevs — APPROVED** [review]

**Pull request merged**